pkg/trace/pb/remote_rates.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+package pb;
+
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+
+message Rate {
+    string service = 1; 
+    string env = 2;
+    double rate = 3; 
```

Comment:
I think that float is enough. Why is this a double?

Reply:
There's no significant gain in picking float32, that's why we use float64 everywhere.

Reply:
Shouldn't rate be called target tps since it's not a sampling rate?

Reply:
@raphaelgavache 
new `proto` is defined here.
https://github.com/DataDog/dd-go/blob/281c4e290216242c2ea29dcfa2c62c546870f530/pb/proto/fleet/product/apm_dynamic_rates.proto